docs/sematic-vs.md

```diff
@@ -56,35 +56,98 @@ But bear with us, we'll catch up fast!
 
 {% endhint %}
 
-### Airflow
+### ... Airflow
 
 Airflow is probably the most popular and widespread way to build end-to-end
-pipelines, since it was the first to provide such abilities.
+pipelines, since it was one of the first to provide such abilities.
 
 Sematic differs from Airflow in the following ways:
 
-* **Iterative development** – change code, run in the cloud, visualize, repeat. In Airflow, you can either run a pipeline on a local Airflow instance, or if you want to run it in the cloud, you must merge your code and deploy it to the cloud instance which adds many steps to your iteration workflow.
+* **Iterative development** – change code, run in the cloud, visualize, repeat. In Airflow, you can either run a pipeline on a local Airflow instance, or if you want to run it in the cloud, you must merge your code and deploy it to the cloud instance which adds many steps to your iteration workflow. In Sematic, you can execute your pipeline entirely locally while you develop: you can even run it
+without running a local server if you want to!
```

Comment:
wanted to add the comparison of what Sematic provides instead. Before this was basically just like "Here's what Airflow doesn't do"

Reply:
added